plugins/action/dtc/get_poap_data.py

```diff
@@ -241,14 +244,19 @@ def run(self, tmp=None, task_vars=None):
                 # fabric setting for this might be modified if/when the create
                 # role is executed later in this run.
                 fail_msg = workflow.refresh_message
-                match_text = r"Please\s+enable\s+the\s+DHCP\s+in\s+Fabric\s+Settings\s+to\s+start\s+the\s+bootstrap"
-                if re.search(match_text, fail_msg, re.IGNORECASE):
-                    pass
-                match_text = r"Invalid\s+Fabric"
-                if re.search(match_text, fail_msg, re.IGNORECASE):
```

Comment:
The bug was here.  We set `match_text` twice so only one of them would ever match.  Instead we need to loop through each possible message we want to ignore.